test/file-store.js

```diff
@@ -0,0 +1,59 @@
+/* global describe, it */
+
```

Comment:
Maybe I missed a few files, but I was trying to add `use strict` to everything.  Can you add it to this new file?